docs/FAQs.md

```diff
@@ -169,3 +169,10 @@ A mar file can be used either locally or be publicly available via http. An S3 U
 
 ### How to set a model's batch size on SageMaker?  Key parameters for TorchServe performance tuning.
 [TorchServe performance tuning example](https://github.com/lxning/torchserve_perf/blob/master/torchserve_perf.ipynb)
+
+## Why is my model initialization so slow?
```

Comment:
Is` model initialization` referring to initializing pytorch model instance and passing the deserialized model state dict from `torch.load( )`?

Reply:
It's really referring to the time it takes for the first inference relative to the second

Reply:
Or is it referring to the end-to-end model loading to torchserve?

Reply:
End to end yeah